packages/components/src/internal/components/editable/utils.test.ts

```diff
@@ -725,9 +726,11 @@ describe('getValidatedEditableGridValue', () => {
             },
             value: '2020-13-23 14:34',
         });
-        expect(getValidatedEditableGridValue(new Date('2020-13-23 14:34'), dateCol).message).toStrictEqual({
-            message: 'Invalid date',
-        });
+        // TODO: If it is a date should we consider it valid? This might revert to an expected failure
+        // with future updates to _formatDate()
```

Comment:
I'm not sure what you mean here. Does the updated code not consider a month value of "13" invalid?

Reply:
I was able to revert any changes needed here as behavior now aligns with the test.